src/_guides/libraries/c-interop.md

```diff
@@ -103,24 +105,40 @@ Here's the code for each step.
 import 'dart:ffi' as ffi;
 ```
 
-2. Create a typedef with the FFI type signature of the C function. <br>
+2. Import the path library that you'll use to store the path of dynamic library.
+```dart
+import 'dart:io' show Platform, Directory;
+import 'package:path/path.dart' as path;
+```
+
+3. Create a typedef with the FFI type signature of the C function. <br>
    Commonly used types defined by dart:ffi library include
    `Double`, `Int32`, `NativeFunction`, `Pointer`, `Struct`, `Uint8`, and `Void`.
 ```dart
 typedef hello_world_func = ffi.Void Function();
 ```
 
-3. Create a typedef for the variable that you'll use when calling the C function.
+4. Create a typedef for the variable that you'll use when calling
+   the C function.
 ```dart
 typedef HelloWorld = void Function();
 ```
 
-4. Open the dynamic library that contains the C function.
+5. Create a variable to store the path of the dynamic library.
+```dart
+  var libraryPath = path.join(Directory.current.path, 'hello_library', 'libhello.so');
+  if (Platform.isMacOS)
+    libraryPath = path.join(Directory.current.path, 'hello_library', 'libhello.dylib');
+  if (Platform.isWindows)
+    libraryPath = path.join(Directory.current.path, 'hello_library', 'Debug', 'hello.dll');
```

Comment:
These lines are really long; we need to shorten them. Also, I think we should follow the style guideline: https://dart.dev/guides/language/effective-dart/style#do-use-curly-braces-for-all-flow-control-structures. And maybe use an `else if`? So maybe this:

```suggestion
var libraryPath = path.join(Directory.current.path, 'hello_library',
    'libhello.so');
if (Platform.isMacOS) { 
  libraryPath = path.join(Directory.current.path, 'hello_library', 
      'libhello.dylib');
} else if (Platform.isWindows) { 
  libraryPath = path.join(Directory.current.path, 'hello_library', 
      'Debug', 'hello.dll');
} 
```

Reply:
👍